src/resources/views/base/inc/scripts.blade.php

```diff
@@ -62,4 +62,15 @@
     $('.nav-tabs a').on('shown.bs.tab', function (e) {
         location.hash = e.target.hash.replace("#tab_", "#");
     });
-</script>
+
+    $(document).ajaxComplete(function(e, result) {
```

Comment:
I think it's important we do this only when `APP_DEBUG=true`

Nobody wants this in production. Probably not in staging either. But I think the best environment variable to tie it to is `APP_DEBUG` - that's usually what's used for this exact purpose right?

Reply:
Fixed 👌